### DISCUSSION TITLE
[UPDATED]DER RIESE

### DISCUSSION THREAD

**Discussion by glitcheking** [nzp-team/nzportable#885] [Map Releases]
Yes yes ! Here it is after some hard work 
DER RIESE full version for pc
The map is fuly working on pc have not tested it on hand held devices 
but i wil make a psp version of it soon!
have fun playing can you find al the hidden teddys to open the secret room
hint teddys wil only spawn if you shoot the button.
find any bugs let me know.

### [download link](https://archive.org/download/derriese_20240825_1215/derriese.zip) <<<
.
.
![Nazi Zombies_ Portable 25_08_2024 05_28_05](https://github.com/user-attachments/assets/7d863412-6574-4d26-b277-1528a916f2c5)

Map FEATURES.
15 wall buy guns
3 teleporters
Traps 
Pap
Easter egg 
4 teddys
Skybox 
Dogs
All perks 
4 player coop
loadscreen picture
menu picture

Future updates.
Custom sounds
Ambience sounds
More props
Second easter egg
Songs 

![Nazi Zombies_ Portable 24_08_2024 12_34_17](https://github.com/user-attachments/assets/f66f721f-df26-423c-a98c-247ed2fcf458)

![Nazi Zombies_ Portable 24_08_2024 12_37_14](https://github.com/user-attachments/assets/9c4f775f-080d-4faf-adc4-6bbea7836d0a)

![Nazi Zombies_ Portable 24_08_2024 12_36_35](https://github.com/user-attachments/assets/6042ad43-520c-44da-a2cc-a8e0fa6d60c8)

![Nazi Zombies_ Portable 24_08_2024 12_39_01](https://github.com/user-attachments/assets/bfb8888c-dd96-45be-9c5f-19a094cecf31)

![Nazi Zombies_ Portable 24_08_2024 12_38_49](https://github.com/user-attachments/assets/df047b0c-e56d-41ab-befe-8a95c06fb6e5)

![Nazi Zombies_ Portable 24_08_2024 12_36_45](https://github.com/user-attachments/assets/1d79aa67-2ae6-4f2d-a1f1-8bff38ad44d7)

![Nazi Zombies_ Portable 24_08_2024 12_37_00](https://github.com/user-attachments/assets/6013b83a-18ba-4a5f-82b5-8b47ed73be02)

![Nazi Zombies_ Portable 24_08_2024 12_37_58](https://github.com/user-attachments/assets/76b1e0a1-c273-4dc0-aeb6-ece03af913be)

**Comment by glitcheking** [comment]
this is a new version i apearently updated the wrong file yesterday and just now found out

**Reply by ToonLink0001** [reply]
ok thanks man!

**Comment by ToonLink0001** [comment]
Hey so I can't find the last teddybear, I shoot the one to start the fly trap, then the furnace, and i can't find the second one, only 3 and 4

**Reply by glitcheking** [reply]
Second room above the zombie spawn on top of the litle roof inside the chimney 
It is possible the secret door wont open now beacause there was a bug with the teddys them selfs

**Reply by ToonLink0001** [reply]
k found them all thanks, looking for the last radio now, found 2

**Reply by ToonLink0001** [reply]
If possible I can send you the beauty of animation ee song to add to the mod pack?

**Reply by glitcheking** [reply]
Sure thats fine i am currently Busy with work i wil be back at it tomorrow and the 3rd radio is not in yet i wil be making a easteregg out of them

**Reply by ToonLink0001** [reply]
k cool, let me know and I'll send the file

**Reply by ToonLink0001** [reply]
i got the ee song file, how you want me to send to you?

**Reply by glitcheking** [reply]
I wil be back at it in a few days contact me on discord

**Reply by ToonLink0001** [reply]
whats your discord?

**Comment by Renatochon** [comment]
Does anyone know how to put those maps on android psp?

**Reply by glitcheking** [reply]
You need to put the files in the psp version of nzp on your phone

**Reply by Renatochon** [reply]
But It's that I put it in the nzp folder then to maps and put it in a dsg file or something like that and it doesn't appear

**Comment by jazzoupy** [comment]
link isnt working with archive.org being down

**Comment by 26lake** [comment]
tysm dude, i love this map

**Reply by 26lake** [reply]
archive.org has been ddosed fuuuck

**Comment by SenorMiggs** [comment]
Hello! Would I be possible if you could send me this file or if you could repost another link to it? The current one seems to be down unfortunately :(

**Reply by jazzoupy** [reply]
not sure why they used an archive instead of a normal download like mediafire

**Reply by MotoLegacy** [reply]
We don't support mediafire or other download services that serve ads or don't have permanent links for direct downloads. This is the first time I have ever witnessed archive.org go down and uptime is historically incredibly reliable. Chill out and wait for them to upgrade their infrastructure.

**Reply by jazzoupy** [reply]
wow an actual update or info of some kind. thanks

**Reply by MotoLegacy** [reply]
It's not my responsibility to provide information on an entirely external source completely unrelated from NZ:P itself. The information on archive.org's status has also been covered via several massive online news fronts. It isn't my problem if you kept yourself in the dark. Happy to help!

**Reply by jazzoupy** [reply]
I was aware of the status of archive.org. Read my other reply? I meant info on the download itself, you know, the thing related to NZ:P. Either way, thanks?

**Comment by ToonLink0001** [comment]
link is down shit

**Comment by PrimiseRichtofen** [comment]
the download doesn't work I've tried everything and did the pk3 trick

**Reply by spectrum1-wave** [reply]
The link is down.

**Comment by Phynix0027** [comment]
The map bugs a lot on nintendo switch, no matter what I touch in the settings

**Comment by irandiaz** [comment]
The truth is you have to fix the map compatibility because it just won't load the PSP.

**Reply by PrimiseRichtofen** [reply]
how do I do that

**Reply by irandiaz** [reply]
Realmente no sé

**Comment by SgtCookieBobble** [comment]
Game has a bug where the dog round fog doesn't disappear which forces to use the buggy non-full bright mode.

This is on the Vita though, so this bug could be considered null as the PSP version isn't out yet (or supported.)

**Comment by patenaudejordan-creator** [comment]
does it work on 3ds

**Comment by patenaudejordan-creator** [comment]
i have it downloaded but it crashes and brings me to a screen with a bunch of text then tells me to quit

**Reply by katniny** [reply]
it won’t boot unless it’s pc, this map is horribly optimized

**Reply by patenaudejordan-creator** [reply]
ok thanks alot

**Reply by kronictripy420** [reply]
it works on Switch HOS 20.5.0 running Atmos very well it just the surrounding walls are glitched like the sky giant black and rainbow rings anyway to ix this?

**Comment by V1cboi123** [comment]
Finally I can play this map without major frames dropin. Map is so far pretty good and works well. (windows Laptop)

**Comment by phatpappa** [comment]
Crashes on old 3DS. Would be awesome if this worked eventually.
<img width="3024" height="4032" alt="unnamed" src="https://github.com/user-attachments/assets/efc7f9c9-3863-453d-88b7-eec6a27f197c" />

**Comment by curtmanps4-gif** [comment]
Does anybody know where all the teddy bear locations are. (Already know people are gonna say I have a skill issue)
